apps/L1G/restaurant/restaurant-frontend/src/components/home/MenuCard.tsx

```
@@ -0,0 +1,42 @@
+import { Sheet, SheetContent, SheetDescription, SheetHeader, SheetTitle, SheetTrigger } from '@/components/ui/sheet';
+export type CardProps = {
+  id: number;
+  image: string;
+  foodName: string;
+  price: string;
+  category: string;
+};
+const MenuCard = ({ image, foodName, price, category }: CardProps) => {
+  const [isSelected, setIsSelected] = useState(false);
+  const [selectCount, setSelectCount] = useState(0);
+  const handleSelect = (id: string) => {
```

ESLint

[error] 'id' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)